pkg/server/admin_cluster_test.go

```diff
@@ -88,7 +88,7 @@ func TestAdminAPITableStats(t *testing.T) {
 	// this to occur, and for full replication.
 	testutils.SucceedsSoon(t, func() error {
 		if err := httputil.GetJSON(client, url, &tsResponse); err != nil {
-			return err
+			t.Fatal(err)
```

Comment:
Why? On the off chance that we do somehow get an error here, wouldn't we legitimately rather retry than deal with a test flake?

Reply:
I'll take my chances. There shouldn't be a way to get a legit error here, should there? It's a common anti pattern to put "too much" into the SucceedsSoon, and I like to clean it up when I drive by. Might regret it. 😄